API.md

```diff
@@ -1643,6 +1643,47 @@ this.player.seek(120, 50); // Seek to 2 minutes with +/- 50 milliseconds accurac
 
 Platforms: iOS
 
+### setPlayerStatus
```

Comment:
Is it possible to remove this api from doc ?
OK to use an internal common function, but each new api shall be maintain, more api is officially available more maintenance is necessarry.
Morover can you rename to : setPlayerPauseState. here it is a too generic name (for internal use)